elementary/monitor/dbt_project/dbt_project.yml

```diff
@@ -29,6 +29,11 @@ clean-targets: # directories to be removed by `dbt clean`
 vars:
   edr_cli_run: true
 
+quoting:
+  database: "{{ env_var('DATABASE_QUOTING', 'None') | as_native }}"
+  schema: "{{ env_var('SCHEMA_QUOTING', 'None')  | as_native }}"
+  identifier: "{{ env_var('IDENTIFIER_QUOTING', 'None') | as_native }}"
```

Comment:
`as_native` is the only way I could think of to be able to pass either True, False or None as values